current/toolkit-walkthrough.md

```diff
@@ -33,11 +33,11 @@ Make a directory in your userspace, somewhere where you can find it: on your des
 
 Use the following command, replacing `/path/to/your/data` with the directory. **If you want to use your own ARC or WARC files, please put them in this directory**.
 
-`docker run --rm -it -v "/path/to/your/data:/data" archivesunleashed/docker-aut:0.50.0`
+`docker run --rm -it -v "/path/to/your/data:/data" archivesunleashed/docker-aut:latest`
 
 For example, if your files are in `/Users/ianmilligan1/desktop/data` you would run the above command like:
 
-`docker run --rm -it -v "/Users/ianmilligan1/desktop/data:/data" archivesunleashed/docker-aut:0.50.0`
+`docker run --rm -it -v "/Users/ianmilligan1/desktop/data:/data" archivesunleashed/docker-aut:latest`
```

Comment:
When running this it pulled down an earlier version of AUT. Is there a command that they should run to force docker to find the most recent image? (when run with `0.50.0` it worked)